static/js/redux/ui/semesterly.tsx

```diff
@@ -222,7 +222,7 @@ const Semesterly = () => {
                 <a href="/privacypolicy">Privacy</a>
               </li>
               <li className="footer-button" role="presentation">
-                <a href="mailto:contact@semester.ly?Subject=Semesterly">Contact us</a>
+                <a href="">semesterly[at]jhu[dot]edu</a>
```

Comment:
This will break the link to automatically create an email to semesterly jhu edu, are you sure you want to do this? People will click on the link and get a 404

Reply:
It won't 404. It'll redirect to the same page.
I might adjust though since it really shouldn't href to anything.